doc/manual/source/advanced-topics/distributed-builds.md

```
@@ -90,12 +90,12 @@ Darwin
 It is possible to specify multiple build machines separated by a semicolon or a newline, e.g.
 
 ```console
-  --builders 'ssh://mac x86_64-darwin ; ssh://beastie x86_64-freebsd'
+  --builders 'ssh://mac aarch64-darwin ; ssh://beastie x86_64-freebsd'
 ```
 
 Remote build machines can also be configured in [`nix.conf`](@docroot@/command-ref/conf-file.md), e.g.
 
-    builders = ssh://mac x86_64-darwin ; ssh://beastie x86_64-freebsd
+    builders = ssh://mac aarch64-darwin ; ssh://beastie x86_64-freebsd
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)